CHANGELOG-3.5.md

```diff
@@ -144,6 +144,7 @@ Note that any `etcd_debugging_*` metrics are experimental and subject to change.
 - [Fix server panic](https://github.com/etcd-io/etcd/pull/12288) when force-new-cluster flag is enabled in a cluster which had learner node.
 - Add [`--self-signed-cert-validity`](https://github.com/etcd-io/etcd/pull/12429) flag to support setting certificate expiration time.
   - Notice, certificates generated by etcd are valid for 1 year by default when specifying the auto-tls or peer-auto-tls option.
+- Log [successful etcd server-side health check in debug level](https://github.com/etcd-io/etcd/pull/12677).
```

Comment:
Can we move this below line 117?

Below

- Log [`[CLIENT-PORT]/health` check in server side](https://github.com/etcd-io/etcd/pull/11704).

Reply:
ack. moved this CHANGELOG entry to line 118.